valen-ui/src/main/java/be/twofold/valen/ui/viewers/data/PreviewValueTreeItem.java

```diff
@@ -53,7 +60,12 @@ private ObservableList<PreviewValueTreeItem> buildChildren(PreviewValueTreeItem
             if (holder.value().getClass().isRecord()) {
                 var value = holder.value();
                 for (RecordComponent recordComponent : value.getClass().getRecordComponents()) {
-                    children.add(new PreviewValueTreeItem(new PreviewItem(recordComponent.getName(), recordComponent.getAccessor().invoke(value))));
+                    try {
+                        children.add(new PreviewValueTreeItem(new PreviewItem(recordComponent.getName(), recordComponent.getAccessor().invoke(value))));
+                    } catch (IllegalAccessException e) {
```

Comment:
Why specifically `IllegalAccessException`? This doesn't sound right. Suppressing errors is bad btw.